go.mod

```diff
@@ -22,3 +22,5 @@ require (
 	gopkg.in/tylerb/graceful.v1 v1.2.15
 	rsc.io/goversion v1.2.0
 )
+
+go 1.13
```

Comment:
Does that mean you need go >= v1.13 ? That would break our policy to support one version below if yes.

Reply:
The go 1.13 tools add that automatically, but go 1.12.10 can still build onet, so I guess it is ok.